src/crypto/dsa.rs

```diff
@@ -41,7 +41,10 @@ impl Signer for SecretKey {
 
         let signature = match hash_algorithm {
             HashAlgorithm::MD5 => signing_key.sign_prehashed_rfc6979::<md5::Md5>(digest),
+
+            // TODO? we can't easily use use sha1_checked here, because it doesn't implement BlockSizeUser
```

Comment:
should be easy to add, want to make a PR for this?

Reply:
see https://github.com/RustCrypto/hashes/pull/582

Reply:
It's merged :tada:

Reply:
nice